src/util/ChartUtils.ts

```diff
@@ -832,7 +841,7 @@ export const offsetSign = (series: any) => {
     for (let i = 0; i < n; ++i) {
       const value = _.isNaN(series[i][j][1]) ? series[i][j][0] : series[i][j][1];
 
-      /* eslint-disable prefer-destructuring */
+      /* eslint-disable prefer-destructuring, no-param-reassign */
```

Comment:
Nice, love how you reduce the scope of the eslint-disable. Great example to follow. ❤️